### PR TITLE
PYTHON-258: Add CONTAINS operator for collections

### PR DESCRIPTION
**Pull request by aboudreault**
I tried to keep this simple and followed what I'm seen in our current code. Perhaps, I didn't forget something else affected by a new operator. If so, let me know.

I indent to add the documentation tomorrow morning.

### CODE REVIEW

File: tests/integration/cqlengine/query/test_queryset.py [hunk 1]
```diff
@@ -516,6 +530,15 @@ def test_indexed_field_can_be_queried(self):
         q = IndexedTestModel.objects(test_result=25)
         assert q.count() == 4
 
+        q = IndexedTestModel.objects.filter(test_list__contains=42)
```
Comment (aholmberg) [inline tests/integration/cqlengine/query/test_queryset.py]: I know the existing codebase uses bare asserts. We didn't revise all tests when we integrated, but I would prefer not to propagate, and usually try to [fix as I touch a given test](https://github.com/datastax/python-driver/commit/a09c7c41ec9df725234279ed9b0350380b3915c2).

Comment (aboudreault) [inline tests/integration/cqlengine/query/test_queryset.py]: Thanks for letting me know. I wasn't sure if there was a preference there.

**Comment by aholmberg** [comment]
+1 with minor (optional) comment on new tests

**Comment by aboudreault** [comment]
comments have been addressed.

**Pull request merged**